eva_cttv_pipeline/trait_mapping/trait.py

```diff
@@ -0,0 +1,65 @@
+class OntologyEntry:
```

Comment:
Could we have tests added for some of these functions, even if they are caused by calls from other files? This file has a 0% coverage.